syft/distro/identify.go

```diff
@@ -145,3 +155,45 @@ func parseBusyBox(contents string) *Distro {
 	}
 	return nil
 }
+
+// example CPE: cpe:/o:centos:linux:6:GA
+var systemReleaseCpeMatcher = regexp.MustCompile(`cpe:\/o:(.*?):.*?:(.*?):.*?$`)
```

Comment:
This is a good approach, however, syft already has CPE parsing built in that we can leverage https://github.com/anchore/syft/blob/main/syft/pkg/cpe.go#L12  --can we switch to using this?

Reply:
@wagoodman Just tried that, but it creates an import cycle :(. Unless i'm doing something horribly wrong?

Reply:
nope, you are not doing anything wrong --it looks like pkg depends on distro to support pURLs. This hints at a refactor that we can do later, either striving for pkg to not depend on distro (medium lift-ish) or migrating cpe out of pkg (small lift). We can defer that for now.

Can you leave a comment in the code that points out we should use the CPE parser once it is importable? (no code change requested at this point 👍 )

Reply:
comment added!